easybuild/easyconfigs/d/DB/DB-18.1.40-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,33 @@
+# EasyBuild easyconfig
+name = 'DB'
+version = '18.1.40'
+
+homepage = 'https://www.oracle.com/technetwork/products/berkeleydb'
+
+description = """Berkeley DB enables the development of custom data management
+ solutions, without the overhead traditionally associated with such custom
+ projects."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+# download via
+# https://www.oracle.com/technetwork/products/berkeleydb/downloads/,
+# requires registration
```

Comment:
@branfosj We can actually auto-download this via `https://download.oracle.com/berkeley-db/db-18.1.40.tar.gz`.

Only works for latest version though, it seems...

Reply:
Better option could be this Homebrew mirror, it does rename the source tarball though (but SHA256 checks out):

```
https://bintray.com/homebrew/mirror/download_file?file_path=berkeley-db-18.1.40.tar.gz
```

Reply:
see also #11454

Reply:
Yes, switching to the Homebrew mirror and having an automatic download is sensible to me. I've done that in d62b2a3.